### PR TITLE
fix(utils): define link alias + pluralize

### PR DESCRIPTION
**Pull request by carlos-r-l-rodrigues**
What:
 * Fix alias of models that have "compound names": eg "related_product"
 * `deleteCascade` option
 * Package to pluralize english words correctly

### CODE REVIEW

**Review by olivermrbl — APPROVED** [review]
LGTM

File: packages/core/utils/package.json [hunk 1]
```diff
@@ -40,6 +40,7 @@
     "dotenv": "^16.4.5",
     "jsonwebtoken": "^9.0.2",
     "knex": "2.4.2",
+    "pluralize": "^8.0.0",
```
Comment (adrien2p) [inline packages/core/utils/package.json:43]: Q: what is the impact with our current pluralize type compared to real output?

**Review by adrien2p — APPROVED** [review]

File: packages/core/utils/package.json (hunk 1 above)
Comment (carlos-r-l-rodrigues) [reply packages/core/utils/package.json:43]: Worst case scenario I will have to fix tests/Interfaces/Modules where we have wrong plurals.
What we might have to revisit is the TypeScript types that won't be correct

Comment (adrien2p) [reply packages/core/utils/package.json:43]: Yes thats definitely my point, the type need to match the function rules, we can check that separately

**Pull request merged**